### ISSUE TITLE
about annotation task

### ISSUE THREAD

**Issue by qkrminho** [bowang-lab/scGPT#305]
Hello,
First of all, thank you for creating such an amazing tool!
I’m reaching out with a few questions regarding the cell annotation process.

1. I’m curious about the differences between zero-shot annotation, reference mapping, and annotation after fine-tuning. As far as I understand, both reference mapping and fine-tuning require reference data with known cell types, but I’m not entirely clear on how the two approaches differ from each other.

2. Second, I’d like to ask whether the reference data required for both reference mapping and fine-tuning-based annotation can simply be any single-cell dataset that contains cell type labels.

3. Lastly, I’m wondering if it’s acceptable to perform fine-tuning using the same dataset that was used during pretraining.

**Comment by subercui** [comment]
Thank you for the questions! 

1. You are right both "reference mapping and fine-tuning require reference data with known cell types". The difference is reference mapping uses the pretrained model directly by extracting cell embeddings from the model, and then transferring the cell type labels from the reference data onto the new cells based on embedding similarities; Fine-tuning, on the other hand, will retrain the model, so it will change how the model computes cell embeddings based on the newly provided data, and there is a classifier to predict cell types rather than label transferring. 

    Practical advice on which to use: given a dataset of interest to be annotated, case 1. if you have a reference data, and you know it contains similar cell types as in the dataset of interest and was sequenced by similar sequencing techniques, then you may try fine-tuning. case 2. if you don't have a "similar enough" reference data, the reference mapping tutorial can be a good option. It will use the cell atlas from cellxgene as reference, and it runs fast.

2. This has been pretty much answered in the above paragraph. Basically, you surely want to use a reference data that are very relevant to the dataset of interest. Also, you can always first try the reference mapping using a reference data of your own or a cell atlas, since the option is faster.

3. Yes, I think it is fine.

**Comment by qkrminho** [comment]
Sorry for asking you again late. 

Based on the last feedback, I checked again in the process of looking at the tutorial, and it seems that the query data also needs a cell type that already exists as ground truth in the last test and evaluation process. 

In fact, I wonder if the cell type should be known in both reference and query when performing cell annotation, or in the case of data in the tutorial, is there a cell type in the queue data for simple performance comparison.

**Comment by subercui** [comment]
Hi, to explain, the workflow doesn't really need annotations from the "query data". It is merely because in the tutorial, we wanted to measure how well the model's predictions align with the ground truth human annotations, so there are the annotation labels in the data for later accuracy evaluation. In your case, if you would like to implement something based on the tutorial notebooks, you may just put in some random annotations in your query data, and then delete or simply ignore the later accuracy evaluation.
